cmd/replay/main.go

```diff
@@ -27,11 +27,14 @@ import (
 )
 
 func getBlockHeaderByHash(hash common.Hash) *types.Header {
-	enc := wavmio.ResolvePreImage(hash)
+	enc, err := wavmio.ResolvePreImage(hash)
+	if err != nil {
+		panic(fmt.Errorf("Error resolving preimage: %w", err))
```

Comment:
Prefer errors.Wrap (it maintains the callstack information)

```suggestion
		panic(errors.Wrap(err, "Error resolving preimage"))
```

Reply:
Fixed, thanks.